META6.json

```diff
@@ -25,5 +26,5 @@
   ],
   "test-depends": [
   ],
-  "version": "0.10"
+  "version": "0.11"
```

Comment:
Don't set version manually, mi6 will take care of it.

Reply:
I don't think that's true for recent versions of App::Mi6

Reply:
It is asking the next release version upon `mi6 release`.

Reply:
Most distributions in community-modules should be using mi6 by now. This would allow to have uniform release process. I have already converted `JSON::Unmarshal`. It is also no distributed via zef ecosystem, thus immediate upping in META6 will have no effect until `mi6 release` is done.